src/jdk.httpserver/share/classes/com/sun/net/httpserver/HttpsServer.java

```diff
@@ -25,80 +25,83 @@
 
 package com.sun.net.httpserver;
 
-import java.net.*;
-import java.io.*;
-import java.nio.*;
-import java.security.*;
-import java.nio.channels.*;
-import java.util.*;
-import java.util.concurrent.*;
-import javax.net.ssl.*;
-import com.sun.net.httpserver.spi.*;
+import java.io.IOException;
+import java.net.BindException;
+import java.net.InetSocketAddress;
+import com.sun.net.httpserver.spi.HttpServerProvider;
 
 /**
- * This class is an extension of {@link HttpServer} which provides
- * support for HTTPS. <p>
- * A HttpsServer must have an associated {@link HttpsConfigurator} object
+ * This class is an extension of {@link HttpServer} which provides support for
+ * HTTPS.
+ *
+ * <p>A {@code HttpsServer} must have an associated {@link HttpsConfigurator} object
  * which is used to establish the SSL configuration for the SSL connections.
- * <p>
- * All other configuration is the same as for HttpServer.
+ *
+ * <p>All other configuration is the same as for {@code HttpServer}.
+ *
  * @since 1.6
  */
 
 public abstract class HttpsServer extends HttpServer {
 
     /**
      */
```

Comment:
If you're going to write a CSR, then you could as well provide the "Constructor for subclasses to call." comment...

Reply:
Comment added. It can be viewed in commit: https://github.com/openjdk/jdk/pull/610/commits/ef5b9fa75886007484f15d74ddf2529f4054ee66